lms/static/js/spec/verify_student/reverify_view_spec.js

```diff
@@ -8,7 +8,6 @@ define(['jquery', 'common/js/spec_helpers/template_helpers', 'js/verify_student/
         describe( 'edx.verify_student.ReverifyView', function() {
 
             var TEMPLATES = [
-                "reverify",
                 "webcam_photo",
```

Comment:
These templates have to be underscore templates (.underscore is appended). There is no reverify.underscore file, though there is a reverify.html file. This was causing console errors, though not actually failing any tests.